### PR TITLE
Render custom organisation logos when provided

### PR DESCRIPTION
**Pull request by fofr**
Goes with:
* https://github.com/alphagov/govuk-content-schemas/pull/514 (tests will fail until merged and deployed)
* https://github.com/alphagov/static/pull/894

Part of https://trello.com/c/HtSGDoRb/594-add-organisation-logos-that-are-images-to-content-item-2

* Pass custom organisation images to component
* Restrict width of component to 25% to match HTML publication
* Include image only when provided
* Don't show custom logos on HTML pubs with many orgs

## HTML pubs
screen shot 2017-02-08 at 15 50 52

## Corp info pages
screen shot 2017-02-08 at 15 51 08

### CODE REVIEW

File: app/presenters/html_publication_presenter.rb [hunk 1]
```diff
@@ -31,4 +31,12 @@ def last_changed
   def organisations
     content_item["links"]["organisations"].sort_by { |o| o["title"] }
   end
+
+  def organisation_logo(organisation)
+    super(organisation).tap do |logo|
```
Comment (gpeng) [inline app/presenters/html_publication_presenter.rb]: You don't need to pass the `organisation` argument here. It happens automagically.

File: test/presenters/html_publication_presenter_test.rb [hunk 2]
```diff
@@ -39,4 +39,25 @@ def format_name
...
     assert presented_item("published").is_a?(OrganisationBranding)
   end
+
+  test 'includes custom organisation logos when a single organisation is listed' do
+    presented = presented_item("updated")
+    organisation = presented.organisations.first
+    example_logo = schema_item("updated")["links"]["organisations"][0]["details"]["logo"]["image"].symbolize_keys
+    presented_logo = presented.organisation_logo(organisation)[:organisation][:image]
+
+    refute presented.organisations.count > 1
```
Comment (gpeng) [inline test/presenters/html_publication_presenter_test.rb]: This hurts my head a bit. `assert count < 2` or `assert_equal 1, presented.organisations.count`?

File: app/presenters/html_publication_presenter.rb (hunk 1 above)
Comment (fofr) [inline app/presenters/html_publication_presenter.rb]: Fixed and rebased

File: test/presenters/html_publication_presenter_test.rb (hunk 2 above)
Comment (fofr) [inline test/presenters/html_publication_presenter_test.rb]: Fixed and rebased

**Pull request merged**